package.json

```diff
@@ -94,11 +97,19 @@
     "unzip": "^0.1.11"
   },
   "devDependencies": {
+    "@babel/cli": "^7.1.5",
+    "@babel/core": "^7.1.6",
+    "@babel/node": "^7.0.0",
+    "@babel/plugin-proposal-class-properties": "^7.1.0",
+    "@babel/plugin-transform-runtime": "^7.1.0",
+    "@babel/polyfill": "^7.0.0",
+    "@babel/preset-env": "^7.1.6",
+    "@babel/preset-react": "^7.0.0",
+    "@babel/runtime-corejs2": "^7.1.5",
     "@types/jest": "^23.1.6",
-    "babel-cli": "^6.26.0",
-    "babel-core": "^6.26.3",
+    "babel-core": "^7.0.0-bridge.0",
```

Comment:
What is the "bridge" version?

Reply:
I had to use it on another project. It's due to fact they've changed the "architecture" of babel (with babel-core) and I think this ease the transition between the old version (that some other plugins depends on) and the new stuff.

Reply:
It turns out it's not necessary :)